src/gsy_e/gsy_e_core/rq_job_handler.py

```diff
@@ -26,19 +26,21 @@ def decompress_and_decode_queued_strings(queued_string: bytes) -> Dict:
 
 def launch_simulation_from_rq_job(scenario: bytes, settings: Optional[Dict],
                                   events: Optional[str], aggregator_device_mapping: str,
-                                  saved_state: bytes, job_id: str):
+                                  saved_state: bytes, job_id: str,
+                                  connect_to_profiles_db: bool = True):
     # pylint: disable=too-many-arguments, too-many-locals
     """Launch simulation from rq job."""
     logging.getLogger().setLevel(logging.ERROR)
     scenario = decompress_and_decode_queued_strings(scenario)
-    gsy_e.constants.CONFIGURATION_ID = scenario.pop("configuration_uuid")
-    if "collaboration_uuid" in scenario:
-        gsy_e.constants.EXTERNAL_CONNECTION_WEB = True
-        GlobalConfig.IS_CANARY_NETWORK = scenario.pop("is_canary_network", False)
-        gsy_e.constants.RUN_IN_REALTIME = GlobalConfig.IS_CANARY_NETWORK
+    if isinstance(scenario, dict):
+        gsy_e.constants.CONFIGURATION_ID = scenario.pop("configuration_uuid")
+        if "collaboration_uuid" in scenario:
+            gsy_e.constants.EXTERNAL_CONNECTION_WEB = True
+            GlobalConfig.IS_CANARY_NETWORK = scenario.pop("is_canary_network", False)
+            gsy_e.constants.RUN_IN_REALTIME = GlobalConfig.IS_CANARY_NETWORK
     saved_state = decompress_and_decode_queued_strings(saved_state)
-    log.error("Starting simulation with job_id: %s and configuration id: %s",
-              job_id, gsy_e.constants.CONFIGURATION_ID)
+    log.info("Starting simulation with job_id: %s and configuration id: %s",
```

Comment:
Sorry, I deleted my old comment. What I really meant is that it is indeed useful to see when a simulation was started even on production, for debugging purposes.

Reply:
Maybe we can change the log level just for this logger? I.e. we show `info` logs (so that we don't need to use `error` to see them on kibana). Not sure if there is some other filter on kibana.

Reply:
> Maybe we can change the log level just for this logger

I guess for this we need to create a separate logger (and do not call `getLogger()`) and change the log level in a central config that does not exist ATM. An overkill IMO. 

> Not sure if there is some other filter on kibana. 
I do not think so.

Reply:
@fievelk I see your PR [here](https://github.com/gridsingularity/gsy-e/pull/1453) now and also see that I gave my LGTM 😅 . I stand by my comment above, that this info to be logged every time is valuable for debugging. There are not that many simulation started AFAIK so the danger of spamming the logs is low.

Reply:
@hannesdiedrich the PR you linked is also changing the log level of the logger:
```py
logging.getLogger().setLevel(logging.WARNING)
```
so it would still log all those messages.
The reason why I didn't merge it, in the end, is that this will spam our logs even more than now.
What I'm suggesting here is a bit different. Instead of changing the log level of the root logger (`getLogger()`) we can change it just for this module:
```py
logger = logging.getLogger(__name__)
logger.setLevel(logging.INFO)  # Or warning
```
AFAIK, this should only change the log level of this logger, not of all loggers. But we should try it.

Reply:
> What I'm suggesting here is a bit different. Instead of changing the log level of the root logger (`getLogger()`) we can change it just for this module:
> 
> ```python
> logger = logging.getLogger(__name__)
> logger.setLevel(logging.INFO)  # Or warning
> ```

Ok, sounds good to me (if it indeed works).

Reply:
@fievelk , It seems that setting level on one logger with a specific name changes all loggers with the name in the program.

Reply:
@mrsaemir could you make an example? The `__name__` variable would contain the full module name, including namespace (e.g. `simulation.graphql.configuration_objects`) so there shouldn't be any other logger with the same name in other modules.

Reply:
@fievelk , I used to think that you mean something like this:
```python
logger = logging.getLogger()
logger.setLevel(logging.INFO)
```

But, then I realized you mean:

```python
logger = logging.getLogger(__name__)
logger.setLevel(logging.INFO)  # Or warning
```

Now, it's fixed as you suggested. Thanks

Reply:
Great! Thanks :) 👍